PURL-TYPES.rst

```diff
@@ -168,6 +168,35 @@ conda
 
       pkg:conda/absl-py@0.4.1?build=py36h06a4308_0&channel=main&subdir=linux-64&type=tar.bz2
 
+cpan
+----
+``cpan`` for CPAN Perl packages:
+
+- The default respository is ``https://www.cpan.org/``.
+- To search CPAN it is recommended to use ``https://metacpan.org``.
```

Comment:
I think we safely can remove this line, since the module search web page isn't relevant for how PackageURLs work.

Reply:
Corrected ;)